lib/index.js

```diff
@@ -3,9 +3,33 @@
  */
 'use strict';
 
+const {agent} = require('bedrock-https-agent');
+const {httpClient} = require('@digitalbazaar/http-client');
 const {JsonLdDocumentLoader} = require('jsonld-document-loader');
 
 const jdl = new JsonLdDocumentLoader();
+
+const webProtocolHandler = {
+  async get({url}) {
+    if(!url.startsWith('http')) {
+      throw new Error('NotFoundError');
+    }
+    let result;
+    try {
+      result = await httpClient.get(url, {agent});
+    } catch(e) {
+      throw new Error('NotFoundError');
+    }
+
+    return {
+      contextUrl: null,
+      document: result.data,
+      documentUrl: url
+    };
+  }
+};
+
 exports.jsonLdDocumentLoader = jdl;
 exports.documentLoader = jdl.documentLoader.bind(jdl);
 exports.JsonLdDocumentLoader = JsonLdDocumentLoader;
+exports.webProtocolHandler = webProtocolHandler;
```

Comment:
`web` should be `http` or `https`, I think. It's not clear what `web` means -- and it means more than `http`/`https`.

Reply:
Maybe there should be a URL handler map -- and you do `urlHandlers.http` and `urlHandlers.https`.

Reply:
K, will change it to httpProtocolHandler. 
(There is a handler map, see usage at https://github.com/digitalbazaar/bedrock-jsonld-document-loader/pull/4/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R73 )

Reply:
@dlongley - renamed it to `httpClientHandler`